core/src/main/java/org/sonarsource/sonarlint/core/rules/RuleDetailsAdapter.java

```diff
@@ -160,7 +160,7 @@ private static String getTabTitle(RuleDetails ruleDetails, String sectionKey) {
   private static String concat(Collection<String> htmlSnippets) {
     return htmlSnippets.stream()
       .filter(StringUtils::isNotBlank)
-      .collect(Collectors.joining("<br/><br/>"));
+      .collect(Collectors.joining());
```

Comment:
@damien-urruty-sonarsource do you remember why you added those extra BR? I think it makes sense to remove them and deal with IDE specific L&F on IDE side (with CSS or something else), but maybe this was required for SLI?

Reply:
FYI this is current display in IntelliJ
![Screenshot 2023-04-18 at 09 34 40](https://user-images.githubusercontent.com/104207951/232705113-919b180d-00a1-47a4-8123-43125b70d0c6.png)

Reply:
No, I don't remember why I added those extra line breaks... Probably not a problem for SLI